lib/ansible/modules/cloud/azure/azure_rm_loadbalancer.py

```diff
@@ -306,26 +306,32 @@
 '''
 
 EXAMPLES = '''
-    # TODO: this example needs update for 2.5+ module args
-    - name: Create a load balancer
-      azure_rm_loadbalancer:
-        name: myloadbalancer
-        location: eastus
-        resource_group: my-rg
-        public_ip: mypublicip
-        probe_protocol: Tcp
-        probe_port: 80
-        probe_interval: 10
-        probe_fail_count: 3
+- name: create load balancer
```

Comment:
could you pls add comments on applied ansible version?

Reply:
on applied ansible version?